JSON-LD Unit tests fail on Windows

In the unit tests for JsonLdExtension, see link below. The URI is asserted against the value: `"file:/tmp/foo/doc.json"` but on Windows the value will include the drive name such as: `"file:/C:/tmp/foo/doc.json"`.

https://github.com/eclipse-edc/Connector/blame/73d53586a6bb82e083de3dbd6b469e006111b39a/extensions/common/json-ld/src/test/java/org/eclipse/edc/jsonld/JsonLdExtensionTest.java#L61C13-L61C13

```java
    @Test
    void verifyCachedDocsFromConfig_oneValidEntry(ServiceExtensionContext context, ObjectFactory factory) throws URISyntaxException {
        Config config = ConfigFactory.fromMap(Map.of(
                "edc.jsonld.document.foo.url", "http://foo.org/doc.json",
                "edc.jsonld.document.foo.path", "/tmp/foo/doc.json")
        );
        context = spy(context);
        when(context.getConfig()).thenReturn(config);
        var extension = factory.constructInstance(JsonLdExtension.class);
        var service = (TitaniumJsonLd) extension.createJsonLdService(context);

        DocumentLoader documentLoader = getFieldValue("documentLoader", service);
        Map<String, URI> cache = getFieldValue("cache", documentLoader);

        assertThat(cache).containsEntry("http://foo.org/doc.json", new URI("file:/tmp/foo/doc.json"));
    }
```

This issue exist across all tests in the specific unit test module. Building in WSL works fine.

Output from one of the unit tests, as example:

```
java.lang.AssertionError: 
Expecting map:
  {"http://foo.org/doc.json"=file:/C:/tmp/foo/doc.json, "http://www.w3.org/ns/odrl.jsonld"=file:/C:/Users/.../Connector/extensions/common/json-ld/build/resources/main/document%5codrl.jsonld}
to contain entries:
  ["http://foo.org/doc.json"=file:/tmp/foo/doc.json]
but the following map entries had different values:
  ["http://foo.org/doc.json"=file:/C:/tmp/foo/doc.json (expected: file:/tmp/foo/doc.json)]
	at org.eclipse.edc.jsonld.JsonLdExtensionTest.verifyCachedDocsFromConfig_multipleEntries_oneIncomplete(JsonLdExtensionTest.java:117)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
```

--- a/extensions/common/json-ld/src/test/java/org/eclipse/edc/jsonld/JsonLdExtensionTest.java
+++ b/extensions/common/json-ld/src/test/java/org/eclipse/edc/jsonld/JsonLdExtensionTest.java
@@ -21,6 +21,7 @@ import org.eclipse.edc.spi.system.configuration.ConfigFactory;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 
+import java.io.File;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.util.Map;
@@ -51,7 +52,7 @@ class JsonLdExtensionTest {
         DocumentLoader documentLoader = getFieldValue("documentLoader", service);
         Map<String, URI> cache = getFieldValue("uriCache", documentLoader);
 
-        assertThat(cache).containsEntry("http://foo.org/doc.json", new URI("file:/tmp/foo/doc.json"));
+        assertThat(cache).containsEntry("http://foo.org/doc.json", new File("/tmp/foo/doc.json").toURI());
     }
 
     @Test
@@ -67,7 +68,7 @@ class JsonLdExtensionTest {
         DocumentLoader documentLoader = getFieldValue("documentLoader", service);
         Map<String, URI> cache = getFieldValue("uriCache", documentLoader);
 
-        assertThat(cache).containsEntry("http://foo.org/doc.json", new URI("file:/tmp/foo/doc.json"));
+        assertThat(cache).containsEntry("http://foo.org/doc.json", new File("/tmp/foo/doc.json").toURI());
     }
 
     @Test
@@ -84,8 +85,8 @@ class JsonLdExtensionTest {
         DocumentLoader documentLoader = getFieldValue("documentLoader", service);
         Map<String, URI> cache = getFieldValue("uriCache", documentLoader);
 
-        assertThat(cache).containsEntry("http://foo.org/doc.json", new URI("file:/tmp/foo/doc.json"));
-        assertThat(cache).containsEntry("http://bar.org/doc.json", new URI("file:/tmp/bar/doc.json"));
+        assertThat(cache).containsEntry("http://foo.org/doc.json", new File("/tmp/foo/doc.json").toURI());
+        assertThat(cache).containsEntry("http://bar.org/doc.json", new File("/tmp/bar/doc.json").toURI());
     }
 
     @Test
@@ -101,7 +102,7 @@ class JsonLdExtensionTest {
         DocumentLoader documentLoader = getFieldValue("documentLoader", service);
         Map<String, URI> cache = getFieldValue("uriCache", documentLoader);
 
-        assertThat(cache).containsEntry("http://foo.org/doc.json", new URI("file:/tmp/foo/doc.json"))
+        assertThat(cache).containsEntry("http://foo.org/doc.json", new File("/tmp/foo/doc.json").toURI())
                 .noneSatisfy((s, uri) -> assertThat(s).isEqualTo("http://bar.org/doc.json"));
 
     }
@@ -119,7 +120,7 @@ class JsonLdExtensionTest {
         DocumentLoader documentLoader = getFieldValue("documentLoader", service);
         Map<String, URI> cache = getFieldValue("uriCache", documentLoader);
 
-        assertThat(cache).containsEntry("http://foo.org/doc.json", new URI("file:/tmp/foo/doc.json"))
+        assertThat(cache).containsEntry("http://foo.org/doc.json", new File("/tmp/foo/doc.json").toURI())
                 .noneSatisfy((s, uri) -> assertThat(s).isEqualTo("http://bar.org/doc.json"));
 
     }

